requirements.txt

```diff
@@ -1,3 +1,5 @@
 coverage
 pytest
 pytest-cov
+numpy
+cython
```

Comment:
These were deliberately left out as they are part of the `install_requires` configuration in the `setuptools` files. But, it makes sense to list _everything_.

Reply:
@duncanmmacleod I've dug a bit more into this, and I think the short answer is that if someone tries to just do:
```
python setup.py install
```
It won't work, or will try to pull dependencies that don't work on python3.6. Do we want to support this mode of install, or make it very clear that `pip install .` should be used. If `pip install .` then I would back out the changes in `requirements.txt`

Reply:
Okay, I added the full contents of "install requires" into here, so that if someone does:
```
pip install -r requirements.txt
python setup.py install
```
it will work. Just doing `setup.py` directly will not work on older python versions as it will pull incompatible versions of scipy/numpy. (I'll stick with using `pip install .`, but I know lots of folks in LVC will try and do this the "wrong" way).